aiida/restapi/translator/calculation/__init__.py

```diff
@@ -28,12 +29,12 @@ class CalculationTranslator(NodeTranslator):
     # A label associated to the present class (coincides with the resource name)
     __label__ = "calculations"
     # The AiiDA class one-to-one associated to the present class
-    from aiida.orm import CalcJobNode
-    _aiida_class = CalcJobNode
+    from aiida.orm import CalculationNode
+    _aiida_class = CalculationNode
     # The string name of the AiiDA class
-    _aiida_type = "process.calculation.calcjob.CalcJobNode"
+    _aiida_type = "process.calculation.calculation.CalculationNode"
```

Comment:
these hard coded strings are all code smells and most likely should be replaced with something else. I remember the other PR that is still open that removes the `_qb_type` but the same goes for `_aiida_type`. It is ok to not address this in this PR, but for future reference that this should most likely be refactored

Reply:
Ok, I'm mentioning @waychal so she also reads this comment.